.github/ISSUE_TEMPLATE

```diff
@@ -1,17 +1,45 @@
 提交 Issue 之前请先阅读 [Issue 指引](https://www.v2ray.com/zh_cn/chapter_01/issue.html)，然后回答下面的问题，谢谢。
 Please answer the following questions before submitting your issue. Thank you.
 
-1) 你正在使用哪个版本的 V2Ray？（如果服务器和客户端使用了不同版本，请注明）
-1) What version of V2Ray are you using?
+1.  你正在使用哪个版本的 V2Ray？（如果服务器和客户端使用了不同版本，请注明）
+    What version of V2Ray are you using(If you are using different version on server and client, please explicitly point out)?
```

Comment:
'What' is preferred, as there are many versions out there and no choice is given. See http://www.bbc.co.uk/worldservice/learningenglish/grammar/learnit/learnitv175.shtml